### DISCUSSION TITLE
This is the first time I have used notify and I have a problem.

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/notify#219] [Q&A]
This is problem:

![error](https://user-images.githubusercontent.com/59057111/218496118-7e8d2a88-8aa1-4648-bd42-1c733652442b.png)

**Comment by ehsandeep** [comment]
@Ezequiel476 you need to share more info about how the notify was used? like input source/cmd used to understand the issue better.

**Reply by ghost** [reply]
Hi, thanks for answer.  Now this message are happen.
![image 03](https://user-images.githubusercontent.com/59057111/221410874-22441ddc-9403-4c15-b5c2-9b821f0fc56e.png)
![image](https://user-images.githubusercontent.com/59057111/221410876-222785b0-c0de-475d-9d57-a7b2c6e128ab.png)
![image 02](https://user-images.githubusercontent.com/59057111/221410879-5e310f88-3e4d-4651-848e-283c7d61e98f.png)

**Reply by ehsandeep** [reply]
@Ezequiel476 Could you share the notify cmd you are running? specifically how / what input you feeding?

**Reply by ghost** [reply]
@ehsandeep if i understand, im in /notify directory. After, write  notify and happen this message. I am still a layman this, sorry.
